adapter/pkg/messaging/connection.go

```diff
@@ -212,3 +231,98 @@ type amqpFailoverURL struct {
 	retryCount      int
 	connectionDelay int
 }
+
+func handleEvent(c *Consumer, key string) error {
+	var err error
+
+	logger.LoggerMsg.Debugf("got Connection, getting Channel for %s events", key)
+	c.Channel, err = c.Conn.Channel()
+	if err != nil {
+		return fmt.Errorf("Channel: %s", err)
+	}
+
+	logger.LoggerMsg.Debugf("got Channel, declaring Exchange (%q)", exchange)
+	if err = c.Channel.ExchangeDeclare(
+		exchange,     // name of the exchange
+		exchangeType, // type
+		true,         // durable
+		false,        // delete when complete
+		false,        // internal
+		false,        // noWait
+		nil,          // arguments
+	); err != nil {
+		return fmt.Errorf("Exchange Declare: %s", err)
+	}
+
+	logger.LoggerMsg.Infof("declared Exchange, declaring Queue %q", key+"queue")
+	queue, err := c.Channel.QueueDeclare(
+		"",    // name of the queue
+		false, // durable
+		true,  // delete when usused
+		false, // exclusive
+		false, // noWait
+		nil,   // arguments
+	)
+	if err != nil {
+		return fmt.Errorf("Queue Declare: %s", err)
+	}
+
+	logger.LoggerMsg.Debugf("declared Queue (%q %d messages, %d consumers), binding to Exchange (key %q)",
```

Comment:
Can we properly format this message. Once it builds seems it does not give a proper meaning